Skuid_Techniques/Multi_Part_Form/Multi_Part_Form.xml

```diff
@@ -0,0 +1,777 @@
+													</styles>
+												</skuid__text>
+												<skuid__table allowColumnFreezing="dragDrop" model="FormData2_List" uniqueid="sk-FzC-40010" mode="edit" showSaveCancel="false" allowSortingByColumn="true">
+													<fields>
+														<field id="TableField2" uniqueid="fi-FzC-40634"/>
+														<field id="TableField3" horizontalAlignment="right" uniqueid="fi-FzC-40635"/>
+														<field id="TableField4" horizontalAlignment="right" uniqueid="fi-FzC-40636"/>
+														<field id="TableField5" horizontalAlignment="right" uniqueid="fi-FzC-40637"/>
+														<field id="TableField6" uniqueid="fi-FzC-40638" displayAs="BUTTON_GROUP"/>
+													</fields>
+													<filtering enableSearch="false"/>
+													<actions>
+														<action type="multi" label="Add">
+															<actions>
+																<action type="createRow" model="FormData2_List" appendorprepend="append" defaultmodefornewitems="edit" affectedrows="context"/>
+															</actions>
+														</action>
+													</actions>
+													<rowActions/>
+													<massActions/>
+													<exportProperties useTableColumns="true"/>
+													<sorting enable="false"/>
+													<styles>
+														<spacing top="4" bottom="4" left="4" right="4"/>
+													</styles>
+												</skuid__table>
+											</components>
+											<styles>
+												<spacing/>
+											</styles>
+											<background type="color" color="#f2f2f2"/>
+										</skuid__wrapper>
+									</components>
+									<buttons>
+										<button label="Back">
+											<actions>
+												<action type="runComponentAction" componentid="sk-2ftV-7225" action="navigateToStep" stepId="step3"/>
+												<action type="updateRow" fieldmodel="UI_Control" affectedrows="context">
+													<updates>
+														<update valuesource="fieldvalue" field="NavPrimary" enclosevalueinquotes="true" value="Three"/>
+													</updates>
+												</action>
+											</actions>
+										</button>
+										<button label="Next" styleSettingsVariant="primary">
+											<actions>
+												<action type="runComponentAction" componentid="sk-2ftV-7225" action="navigateToStep" stepId="step5"/>
+												<action type="updateRow" fieldmodel="UI_Control" affectedrows="context">
+													<updates>
+														<update valuesource="fieldvalue" field="NavPrimary" enclosevalueinquotes="true" value="Five"/>
+													</updates>
+												</action>
+											</actions>
+										</button>
+									</buttons>
+								</step>
+								<step stepId="step5" stepLabel="Step 5">
+									<components>
+										<skuid__wrapper uniqueid="sk-G0t-28420" heightStrategy="fixed" height="60">
+											<components>
+												<skuid__text contents="Step 5.&amp;nbsp; &amp;nbsp;{{Sec5Summary}} Assets added.&amp;nbsp;&amp;nbsp;" uniqueid="sk-G0t-28421" styleSettingsVariant="increased" model="UI_Control">
```

Comment:
```suggestion
												<skuid__text contents="Step 5.&amp;nbsp; &amp;nbsp;{{Sec5Summary}} assets added.&amp;nbsp;&amp;nbsp;" uniqueid="sk-G0t-28421" styleSettingsVariant="increased" model="UI_Control">
```

One last capitalization thing. Gotta make it nice since it's in SamplePages!